data/assets/examples/pointclouds/points.asset

```diff
@@ -100,13 +100,17 @@ local FixedColor_ScaleBasedOnData = {
       FixedColor = { 0.5, 0.5, 0.0 }
     },
     SizeSettings = {
-      -- The options for the columns that the points can be scaled by. The first
-      -- alternative is chosen per default
-      SizeMapping = { "number_withNan", "a" },
+      SizeMapping = {
+        -- The options for the columns that the points can be scaled by. The first
+        -- alternative is chosen per default
+        ParameterOptions = { "number_withNan", "a" },
```

Comment:
As the dataset used is an example dataset, it has values to show different features. This column name reflects the fact that the data column in the dataset includes missing value (NaN values). The purpose is to showcase the color mapping feature related to missing values  color mapping (they can be shown in a separate color)

I can see how it's a bit confusing for this specific case, since it is used for size mapping. perhaps it would be better to use the b column instead here?

Reply:
I switched which columns are used, to avoid confusion